frontend/cypress/integration/nodes-page.spec.js

```diff
@@ -9,14 +9,13 @@ context("Nodes", () => {
 
   it("Check for node page", () => {
     cy.url().should("include", "/nodes/validators");
-    cy.get("#validator-node").find(".node-selector").contains("Validating");
-    cy.get("#online-node").find(".node-selector").contains("Online-nodes");
-    cy.get("#proposal-node").find(".node-selector").contains("Proposal-nodes");
+    cy.get("#validator-node").contains("Validating");
+    cy.get("#online-node").contains("Online-nodes");
+    cy.get("#proposal-node").contains("Proposal-nodes");
   });
 
   it("Check validators tab", () => {
     cy.url().should("include", "/nodes/validators");
-    cy.get("#validator-node").should("have.class", "node-selected");
```

Comment:
Can we update the test instead of just removing it?

Reply:
sure